strategies.py

```diff
@@ -1,6 +1,4 @@
-"""
-Some example strategies for people who want to create a custom, homemade bot.
-"""
+"""Some example strategies for people who want to create a custom, homemade bot."""
```

Comment:
Add sentence at end: "With these classes, bot makers will not have to implement the UCI or XBoard interfaces themselves."